apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -123,7 +121,39 @@ async function canManageTickets ({ authentication: { item: user }, operation, it
             deletedAt: null,
         })
 
-        return residents.length > 0
+        if (residents.length === 0) {
+            return false
+        }
+
+        if (operation === 'create') {
```

Comment:
Looks like resident can create any ticket to any organization

Reply:
No, on line 126, if there are no residents by property, false is returned.
Added test for this case